### PR TITLE
Remove unused import and add version to deprecations.

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: jupyterhub/app.py [hunk 1]
```diff
@@ -464,7 +462,8 @@ def _cookie_secret_check(self, change):
 
     @observe('api_tokens')
     def _deprecate_api_tokens(self, change):
-        self.log.warning("JupyterHub.api_tokens is pending deprecation."
+        self.log.warning("JupyterHub.api_tokens is pending deprecations"
```
Comment (willingc) [inline jupyterhub/app.py]: Strike the `s` at the end of deprecations. Should read `deprecation` :-)

File: jupyterhub/app.py [hunk 2]
```diff
@@ -1163,8 +1162,7 @@ def init_oauth(self):
         self.oauth_provider = make_provider(
             self.session_factory,
             url_prefix=url_path_join(base_url, 'api/oauth2'),
-            login_url=url_path_join(base_url, 'login')
-,
+            login_url=url_path_join(base_url, 'login'),
```
Comment (willingc) [inline jupyterhub/app.py]: I don't know that we technically need a comma here as `make_provider` only takes 3 arguments.

File: jupyterhub/app.py (hunk 1 above)
Comment (Carreau) [inline jupyterhub/app.py]: Oops, don't know why a s was added.

File: jupyterhub/app.py (hunk 2 above)
Comment (Carreau) [inline jupyterhub/app.py]: Good point.

**Comment by Carreau** [comment]
> I'm assuming you added the versions to make things more user friendly :-)

And to be more confident in when we can delete code. I like to delete code :-)

**Comment by willingc** [comment]
Thanks @Carreau 👍

**Pull request merged**